ReadingJavaScript/README.md

```diff
@@ -0,0 +1,37 @@
+
+##  Basic Literal Types types in Javascript:
+
+* **Null** - its basically a zero or no value. 
```

Comment:
Can you expand and clarify on `null`? Is it really zero? Check this out: [MDN: null](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/null).